### PR TITLE
Warhead EventArgs renamed

### PR DESCRIPTION
**Pull request by ClaudioPanConQueso**

### CODE REVIEW

**Comment by ClaudioPanConQueso** [comment]
For exiled 6

**Review by louis1706 — APPROVED** [review]

**Comment by joker-119** [comment]
Since these are already in the Warhead namespace, is this even necessary? 
afaik there aren't even other events that share the same names, right?

File: Exiled.Events/Handlers/Scp079.cs [hunk 1]
```diff
@@ -143,7 +143,7 @@ public static void OnStartingSpeaker(StartingSpeakerEventArgs ev)
         /// <summary>
         ///     Called before SCP-079 finishes using a speaker.
         /// </summary>
-        /// <param name="ev">The <see cref="StoppingEventArgs" /> instance.</param>
+        /// <param name="ev">The <see cref="StoppingWarheadEventArgs" /> instance.</param>
         public static void OnStoppingSpeaker(StoppingSpeakerEventArgs ev)
```
Comment (IRacle1) [inline Exiled.Events/Handlers/Scp079.cs:147]: xD, fix it

**Review by IRacle1 — CHANGES_REQUESTED** [review]

**Comment by IRacle1** [comment]
> Since these are already in the Warhead namespace, is this even necessary? 
> afaik there aren't even other events that share the same names, right?

Yes, but most of people search events by EventArg name, it will be easier for them and for understanding that event + also many event are named with StoppedXXX

**Comment by ClaudioPanConQueso** [comment]
First time i was coding, i didn't know that StartingEventArgs and Stopping was from the warhead.

File: Exiled.Events/Handlers/Scp079.cs (hunk 1 above)
Comment (ClaudioPanConQueso) [reply Exiled.Events/Handlers/Scp079.cs:147]: What thing? XD

Comment (ClaudioPanConQueso) [reply Exiled.Events/Handlers/Scp079.cs]: fixed

**Comment by Thundermaker300** [comment]
It is very much necessary

**Review by Michal78900 — APPROVED** [review]

**Review by xNexus-ACS — APPROVED** [review]

**Comment by zhaguanyang** [comment]
I would say the idea is fine, but you forgot to rename it in patches. You should actually build it before trying to PR.

**Review by Michal78900 — CHANGES_REQUESTED** [review]
Fix build issues please

**Comment by ClaudioPanConQueso** [comment]
> Fix build issues please

ok

**Comment by ClaudioPanConQueso** [comment]
done

**Pull request closed without merging**